stream_builder/lib/main.dart

```diff
@@ -1,93 +1,82 @@
-          ),
-        ));
+    return new MaterialApp(
+      title: 'Flutter Demo',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: new MyHomePage(),
+    );
   }
 }
 
-class MyStatefulWidget extends StatefulWidget {
-  MyStatefulWidget({Key key}) : super(key: key);
-
+class MyHomePage extends StatefulWidget {
   @override
-  _MyStatefulWidgetState createState() => _MyStatefulWidgetState();
+  _MyHomePageState createState() => new _MyHomePageState();
 }
 
-class _MyStatefulWidgetState extends State<MyStatefulWidget> {
-  Future<String> _calculation =
-      Future.delayed(Duration(seconds: 2), () => 'Large Latte');
+class _MyHomePageState extends State<MyHomePage> {
 
+  @override
   Widget build(BuildContext context) {
-    return DefaultTextStyle(
-      style: Theme.of(context).textTheme.headline2,
-      textAlign: TextAlign.center,
-      child: FutureBuilder<String>(
-        //////////////////////
-        future: _calculation, // a previously-obtained Future<String> or null
+    var futureBuilder = new FutureBuilder(
+      future: _getData(),
+      builder: (BuildContext context, AsyncSnapshot snapshot) {
+        switch (snapshot.connectionState) {
+          case ConnectionState.none:
+          case ConnectionState.waiting:
+            return new Text('loading...');
+          default:
+            if (snapshot.hasError)
+              return new Text('Error: ${snapshot.error}');
+            else
+              return createListView(context, snapshot);
+        }
+      },
+    );
 
-        builder: (BuildContext context, AsyncSnapshot<String> snapshot) {
-          List<Widget> children;
-          if (snapshot.hasData) {
-            children = <Widget>[
-              Image(
-                image: NetworkImage(
-                    'https://globalassets.starbucks.com/assets/b635f407bbcd49e7b8dd9119ce33f76e.jpg?impolicy=1by1_wide_1242'),
-                width: 600,
-                height: 300,
-              ),
-              Padding(
-                padding: const EdgeInsets.only(top: 16),
-                child: Text('Order 66 : ${snapshot.data}'),
-              )
-            ];
-          } else if (snapshot.hasError) {
-            children = <Widget>[
-              Icon(
-                Icons.error_outline,
-                color: Colors.red,
-                size: 60,
-              ),
-              Padding(
-                padding: const EdgeInsets.only(top: 16),
-                child: Text('Error: ${snapshot.error}'),
-              )
-            ];
-          } else {
-            children = <Widget>[
-              SizedBox(
-                child: CircularProgressIndicator(),
-                width: 60,
-                height: 60,
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text("Home Page"),
+      ),
+      body: futureBuilder,
+    );
+  }
+
+  Future<List<String>> _getData() async {
+    var values = new List<String>();
```

Comment:
In Dart, you don't have to use `new`. Please remove it from this line and all the other lines.